dev-requirements.in

```diff
@@ -2,6 +2,7 @@
 
 black
 coverage[toml]
+doltcli
```

Comment:
Can we remove this?

Reply:
would a test-dependency in the plugin pull it in?

Reply:
I guess I'm a little confused... you're saying that users of the sqlalchemy plugin will need to install dolt dependencies as well to use it?  If that's the case, can we just move this line into the sqlalchemy setup.py?

Reply:
sqlalchemy is database-independent, but needs at least one for testing. Moving to the sqlalchemy setup is OK but it's a test dependency -- if it's in the sqlalchemy setup.py test deps will it actually be installed for tests? (I think no? but I don't completely understand the micro-package organization.) I probably should have done this all with sqlite, it will take me awhile to rewrite the tests though